lib/publiccloud/provider.pm

```diff
@@ -490,6 +491,7 @@ sub terraform_apply {
     }
 
     # Return an ARRAY of objects 'instance'
```

Comment:
comment should stay above return . Or you may just delete it because it is stating the obvious so does not hold any real information ...

Reply:
I did , you insert `publiccloud::instances::set_instances(@instances);` between comment and the line which it belongs too . So from now on reader will assume that comment corresponds to `publiccloud::instances::set_instances(@instances);` which does not make any sense